contracts/test/core/invoke/Issuance.t.sol

```diff
@@ -4,8 +4,53 @@ import {StatefulTest} from "core-test/State.t.sol";
 import {TokenInfo} from "src/Common.sol";
 import {IERC20} from "forge-std/interfaces/IERC20.sol";
 import {SCALAR} from "src/lib/FixedPoint.sol";
+import {IVault} from "src/interfaces/IVault.sol";
 
 contract IssuanceTest is StatefulTest {
+    function testGoToZero() public {
+        seedInitial(10);
+        mint(5e18);
+        burn(indexToken.totalSupply());
+        address[] memory underlying = vault.underlying();
+        assertEq(indexToken.totalSupply(), 0);
+        for (uint256 i; i < underlying.length; i++) {
+            assertEq(IERC20(underlying[i]).balanceOf(address(vault)), 1);
+        }
+    }
+
+    function testCantGoToZeroWithUnmintedInflation() public {
+        // Q: Who keeps the intraday fee on the tokens that are being burnt? The user, vault, or the feeRecipient?
+        // A: For now, the vault keeps the intraday fee on the tokens that are being burnt.
```

Comment:
is this true? `realUnits` includes intraday, which is used for redemption, so I also dont understand this conclusion